packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -180,4 +156,69 @@ export class HttpServer extends RunnableModule {
 
     return serverClosePromise(this.server);
   }
+
+  // eslint-disable-next-line sonarjs/cognitive-complexity
+  private async initMetrics() {
+    const promRegistry = new Registry();
+    const anyService = this.#dependencies.services[0];
+    // eslint-disable-next-line no-new
+    new Gauge<string>({
+      async collect() {
+        const currentValue = (await anyService.healthCheck()).localNode?.networkSync;
+        if (currentValue) this.set(currentValue);
+      },
+      help: 'Node synchronization status',
+      name: 'node_sync_percentage',
+      registers: [promRegistry]
+    });
+    for (const service of this.#dependencies.services) {
+      // '-' replaced with '_' to satisfy name validation rules
+      const serviceName = service.slug.replace('-', '_');
+      // eslint-disable-next-line no-new
+      new Gauge<string>({
+        async collect() {
+          const data = await service.healthCheck();
+          let syncStatus;
+          if (data.projectedTip && data.localNode?.ledgerTip) {
+            syncStatus =
+              (data.projectedTip.blockNo * 100) / (data.localNode?.ledgerTip?.blockNo - DB_BLOCKS_BEHIND_TOLERANCE);
+          }
+          if (syncStatus) {
+            if (syncStatus > 100) this.set(100);
```

Comment:
This probably means that a rollback happened and projector hasn't caught up yet. Would be best to not show it as 100%
```suggestion
            if (syncStatus > 100) this.set(syncStatus - 100);
```

Reply:
The intention is to show 100% sync status if it is within DB_BLOCKS_BEHIND_TOLERANCE range.
When `projectedTip.blockNo == ledgerTip.blockNo` the formula above will show result slightly more than 100, so `if (syncStatus > 100) this.set(100);` just ensures that reslut will 100% when projectedTip is somewhere in `(ledgerTip - DB_BLOCKS_BEHIND_TOLERANCE <=  projectedTip <= ledgerTip)`

Reply:
Oh I see, ok. It seems like an odd behavior to me. Why not just report the truth (the 99.9x%)?